stl/inc/regex

```diff
@@ -1710,7 +1710,7 @@ private:
     bool _OctalDigits();
     void _Do_ex_class(_Meta_type);
     bool _CharacterClassEscape(bool);
-    _Prs_ret _ClassEscape(bool);
+    _Prs_ret _ClassEscape2();

```

Comment:
I know, I just want to avoid that someone adds a boolean parameter again later. If this is `_ClassEscape2()`, it's unlikely some later change will switch back to `_ClassEscape(bool)`.

The parser and matcher are complex pieces of code, so I will be relatively aggressive about renaming methods in the next PRs as well to mostly eliminate any mix-and-match scenarios. This especially applies to the matcher: We really don't need any inscrutable bug reports because `_Matcher::_Match()` executes old code while `_Matcher::_Skip()` picks up new code or vice versa.